.github/workflows/terragrunt-apply-production.yml

```diff
@@ -68,7 +68,7 @@ jobs:
       - name: Configure AWS credentials using OIDC
         uses: aws-actions/configure-aws-credentials@e3dd6a429d7300a6a4c196c26e071d42e0343502 # v4.0.2
         with:
-          role-to-assume: arn:aws:iam::${{ env.AWS_ACCOUNT_ID }}:role/forms-terraform-apply-release
```

Comment:
So, what is going to happen to the `-release` role? Will it still be used somewhere? How come we did not have a similar issue in staging?how come we did not have a similar issue in staging?

Reply:
Once the `3.10.1` release goes in, I'll get a fix in to remove the `-release` role since we won't need it anymore.  We didn't have this problem in Staging because its TF apply workflow has always executed on pushes to `develop`, which is what the default SRE OIDC `-apply` roles are setup to work with.

We needed the `-release` role for Prod because it used to execute against a tag reference.  However, with the latest workflow change to support easier revert PRs, it now executes against `develop` as well.